git-new.ps1

```diff
@@ -1,5 +1,21 @@
 #!/usr/bin/env pwsh
 
+<#
+.SYNOPSIS
+    Creates a new branch and checks it out from the specified branches
+
+.PARAMETER branchName
+    Specifies the name of the branch.
+
+.PARAMETER comment
+    Specifies a comment as part of the commit message for the upstream branch.
+
+.PARAMETER upstreamBranches
+    A comma-delimited list of branches (without the remote, if applicable). If not specified, assumes the default service line (see [tool-config](./tool-config.md).)
```

Comment:
Do we want to add "usually main" to this?

e.g.
```
If not specified, assumes the default service line (see [tool-config](./tool-config.md).), usually main
```

Reply:
I only reformatted from the current docs, and there's been no questions on those. I'd rather not scatter cross-linking information like that throughout the docs, though, since that could change and I don't want anything in the docs to be out of date.